src/client/airplanedocgenerator/src/views/NewProjectScreen.js

```diff
@@ -0,0 +1,104 @@
+import React from "react";
+
+import clouds from "../assets/misc/images/cloud.jpg";
+import Button from "../assets/components/Button";
+
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome'
+import { faQuestionCircle } from "@fortawesome/free-solid-svg-icons";
+
+/**
+ * Classe que representa a página de criação de um projeto
+ * @author Bárbara Port
+ */
+class NewProjectScreen extends React.Component {
+     constructor() {
+          super();
+
+          this.cardTitle = "Crie um novo projeto de manual";
+          this.cardText = "Esta sessão irá lhe auxiliar na criação de um novo projeto de manual. Preencha os campos e siga as instruções.";
+     }
+
+     /**
+      * Prevenção de imagens arrastáveis
+      * @param {Event} event Evento recebido na tentativa de arrastar uma imagem
+      * @author Rafael Furtado
+      */
+     preventImageDrag(event) {
+          event.preventDefault();
+     }
+
+     /**
+      * Imagem de fundo
+      * @returns Elemento com a imagem de fundo
+      * @author Rafael Furtado
+      */
+     getBackgroundImage() {
+          let backgroundImage =
+               <img src={clouds} alt="Clouds"
+                    onDragStart={this.preventImageDrag}
+                    className="w-full h-full select-none absolute filter blur-blurLogin opacity-20" />
+
+          return backgroundImage;
+     }
+
+     /**
+      * Função que cria a página
+      * @author Bárbara Port
+      * @returns Página de criar um novo projeto
+      */
+     getNewProjectScreen() {
+          let newProjectScreen = (
+               <div id="contentDisplay" className="w-full h-full">
+                    {this.getBackgroundImage()}
+                    <div id="newProjectScreen" className="w-screen h-screen flex justify-center items-center">
+                         <div className="bg-white lg:w-5/12 md:w-10/12 h-5/6 relative flex justify-start text-center flex-col border-r-8 border-b-8 border-accent">
+                              <div className="m-8">
+                                   <p className="text-2xl font-bold">Crie um novo projeto de manual</p>
+                                   <hr></hr>
+                                   <p className="text-lg mt-4">Esta sessão irá lhe auxiliar na criação de um novo projeto de manual, preencha os campos e siga as instruções!</p>
+                              </div>
+                              <form>
```

Comment:
Bárbara, falta colocar a função do atributi "onSubmit" do form

Reply:
basta adicionar: onSubmit={this.suaFuncao}

Ai vc cria uma função na classe que receba como parametro "event", qualquer dúvida olhar no form da página de login

Reply:
Acho que não dá pra receber um evento como parâmetro. Dá erro.